.github/pull_request_template.md

```diff
@@ -16,9 +17,22 @@ Please complete this section if any breaking changes have been made, otherwise d
 
 - [ ] I have assigned this PR to myself
 - [ ] I have added at least 1 reviewer
-- [ ] I have tested this code
-- [ ] I have updated the README and other relevant documents (guides...)
-- [ ] I have added sufficient documentation both in code, as well as in the READMEs
+- [ ] I have added the relevant labels
+- [ ] I have updated the official documentation
+- [ ] I have added sufficient documentation in code
+
+## Testing
+
+- [ ] I ran the official test suite
+- [ ] I ran custom tests
+
+### Custom tests
```

Comment:
Similar to above comment from Lazar, i would rename this to "Steps to reproduce manual testing"